.github/workflows/jdbc-tests-pgaudit-enable.yml

```diff
@@ -0,0 +1,191 @@
+        with:
+          name: postgres-log-jdbc
+          path: |
+            ~/psql/data/logfile
+            ~/psql/data_5433/logfile
+
+      # The test summary files contain paths with ':' characters, which is not allowed with the upload-artifact actions
+      - name: Rename Test Summary Files
+        id: test-file-rename
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        run: |
+          cd test/JDBC/Info
+          timestamp=`ls -Art | tail -n 1`
+          cd $timestamp
+          mv $timestamp.diff ../output-diff.diff
+          mv "$timestamp"_runSummary.log ../run-summary.log
+          cd ..
+          # get the replication output diff as well if it is present
+          dir_count=`ls | wc -l`
+          if [[ $dir_count -eq 2 ]];then
+            timestamp=`ls -rt | tail -n 2 | sort -r | tail -n 1`
+            cd $timestamp
+            mv $timestamp.diff ../replication-output-diff.diff
+            mv "$timestamp"_runSummary.log ../replication-run-summary.log
+
+      - name: Upload Run Summary 
+        if: always() && steps.test-file-rename.outcome == 'success'
+        uses: actions/upload-artifact@v3
+        with:
+          name: run-summary.log
+          path: |
+            test/JDBC/Info/run-summary.log
+            test/JDBC/Info/replication-run-summary.log
+
+      - name: Upload Output Diff
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        uses: actions/upload-artifact@v3
+        with:
+          name: jdbc-output-diff.diff
+          path: |
+            test/JDBC/Info/output-diff.diff
+            test/JDBC/Info/replication-output-diff.diff
+
+      - name: Check and upload coredumps
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        uses: ./.github/composite-actions/upload-coredump
+
+      - name: Generate Code Coverage
+        id: generate-code-coverage
+        if: always() && (steps.jdbc.outcome == 'success' && steps.replication.outcome == 'success')
+        run: |
+          export PG_CONFIG=~/psql/bin/pg_config
+          export PG_SRC=~/work/postgresql_modified_for_babelfish
+          export cmake=$(which cmake)
+          cd contrib
+          for ext in babelfishpg_common babelfishpg_money babelfishpg_tds babelfishpg_tsql  
+          do
+            cd $ext
+            /usr/bin/lcov --gcov-tool /usr/bin/gcov -q --no-external -c -d . -d ./ -o lcov_test.info
+            cd ..
+          done
+        shell: bash
+
+      - name: Summarize code coverage
+        id: code-coverage-summary
+        if: always() && steps.generate-code-coverage.outcome == 'success'
+        run: |
+          cd contrib/
+          lcov -a babelfishpg_tsql/lcov_test.info -a babelfishpg_tds/lcov_test.info -a babelfishpg_common/lcov_test.info -a babelfishpg_money/lcov_test.info -o jdbc-lcov.info
+          lcov --list jdbc-lcov.info
+
```

Comment:
we can remove all the code coverage related steps from this workflow

Reply:
You can refer to `jdbc-tests-with-parallel-query.yml` to find out which steps are not needed